packages/firebase_auth/firebase_auth/ios/Classes/FLTFirebaseAuthPlugin.m

```diff
@@ -1336,6 +1399,17 @@ - (void)signInWithProviderApp:(nonnull AuthPigeonFirebaseApp *)app
                                                 FlutterError *_Nullable))completion {
   FIRAuth *auth = [self getFIRAuthFromAppNameFromPigeon:app];
 
+  if ([signInProvider.providerId isEqualToString:kSignInMethodGameCenter]) {
+    completion(
+        nil,
+        [FlutterError
+            errorWithCode:@"sign-in-failure"
+                  message:
+                      @"Game Center sign-in requires signing in with 'linkWithCredential()' API."
```

Comment:
```suggestion
                      @"Game Center sign-in requires signing in with 'signInWithCredential()' API."
```

Reply:
Should we have the same error for linkWithProvider?

Reply:
Yes, just double checked, it's the same error response as signing in with provider:
```bash
"[auth/operation-not-allowed] - The federated login request is not supported."
```

I'll update the code 👍

Reply:
Done 👍